packages/-ember-data/tests/integration/record-data/record-data-state-test.ts

```diff
@@ -127,11 +125,6 @@ module('integration/record-data - Record Data State', function(hooks) {
     let { owner } = this;
 
     class LifecycleRecordData extends TestRecordData {
-      constructor(sw) {
-        super();
-        storeWrapper = sw;
```

Comment:
This one was a little weird. `storeWrapper` is not actually being used anywhere so this was triggering the `@typescript-eslint/no-unused-vars` rule.